### ISSUE TITLE
Search everything returns error http 400

### ISSUE THREAD

**Issue by plantarum** [egh/zotxt-emacs#41]
Hi,

I'm using the latest version, including the new commit to support searching all groups. It solves my previous issue accessing group libraries. However, if I try to search "everything" it fails with the error `(error http 400)`. The same search with the options `title, creator, year` or `fields` works as expected.

Thanks again,

Tyler

zotxt-emacs 8a88de5 master origin/master Group together custom groups
GNU Emacs 26.3 (build 1, x86_64-pc-linux-gnu, GTK+ Version 3.24.12) of 2019-11-26
Org mode version 9.3.6 (9.3.6-69-g3bbaca-elpaplus @ /home/tws/.emacs.d/elpa/org-plus-contrib-20200511/)

**Comment by egh** [comment]
Sorry for the late response. Is this still an issue?

**Comment by plantarum** [comment]
I think so. I've since changed my workflow so that it doesn't effect me anymore, though. 

I learned that Better BibTeX supports syncing my Zotero library to both BibTeX and CSL files, so I'm doing that now. That lets me use ivy-bibtex for cite-as-you-write support, using the .bib file to search and insert citation keys. Which means I'm not using zotxt at all in this process.

It would be nice to completely replace my bibtex system, and pull citations directly from Zotero via zotxt. But at the moment, this is working for me.

Thanks!
